config.py

```diff
@@ -1,29 +1,31 @@
 import os
-import sys
-from typing import Optional
 
-from ape import networks, accounts
+from ape import accounts
 
 
-def get_network_name() -> Optional[str]:
-    # It is None at moment of config.py network initialization
-    if networks.active_provider is None:
-        network_index = sys.argv.index("--network")
-        if network_index + 1 < len(sys.argv):
-            network_arg = sys.argv[network_index + 1]
-            _, name, provider = network_arg.split(":")
-            return name
-        return None
+NETWORK_ENV_VAR = "NETWORK"
+ALLOWED_NETWORKS = ["mainnet", "goerli", "holesky"]
 
-    return networks.active_provider.network.name
 
+def get_network_name() -> str:
+    network_name = os.environ.get(NETWORK_ENV_VAR, None)
+    if network_name not in ALLOWED_NETWORKS:
+        raise RuntimeError(f"Must specify env variable {NETWORK_ENV_VAR} be one of {','.join(ALLOWED_NETWORKS)}")
 
-if get_network_name() in ("goerli", "goerli-fork"):
+    return network_name
+
+
+if get_network_name() == "goerli":
     print(f"Using config_goerli.py addresses")
     from config_goerli import *
-else:
+elif get_network_name() == "mainnet":
     print(f"Using config_mainnet.py addresses")
     from config_mainnet import *
+elif get_network_name() == "holesky":
+    print(f"Using config_holesky.py addresses")
+    from config_holesky import *
+else:
+    assert False, f"Unknown network {get_network_name()}"
```

Comment:
We can avoid calling the function by storing the name in a var.

network = get_network_name()

if network == GOERLI:
    # ...
    
P.S. Python 3.10 introduced pattern matching (switch statements), which improves readability, but I see that the minimal version for this project is 3.8.